src/test/StdAssertions.t.sol

```diff
@@ -0,0 +1,368 @@
+// SPDX-License-Identifier: Unlicense
+pragma solidity 0.8.10;
+
+import "../Test.sol";
+
+contract StdAssertionsTest is Test
+{
+    // todo: figure out how to expectEmit multiple events
```

Comment:
I think the `assertEq` overloads are simple enough that we don't really need tests for them as they may add more clutter than value—even the original ones don't have tests 😅

But you can stack expect emits like this:

```solidity
vm.expectEmit(true, true, true, true);
emit MyFirstEvent(val1);
vm.expectEmit(true, true, true, true);
emit MySecondEvent(val2);

// This will look for both events in the below call
myContract.myCall();
```

Reply:
Thanks for the expectEmit answer! Re testing assertEq im flexible, it was quite easy to just template all these tests and fill them in (some of it was even macro-able). If you think their value as regression guards is outweighed by the clutter (given it would be impressive to regress assertEq()), then okay for them to be dropped

Reply:
Hmm, paging @ZeroEkkusu for a second opinion here since I don't feel too strongly

Reply:
DSTest's convention being to not have a corresponding test file for any assertions?